src/java/org/apache/cassandra/tools/nodetool/DisableAutoCompaction.java

```diff
@@ -17,25 +17,36 @@
  */
 package org.apache.cassandra.tools.nodetool;
 
-import io.airlift.airline.Arguments;
-import io.airlift.airline.Command;
-
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 
 import org.apache.cassandra.tools.NodeProbe;
-import org.apache.cassandra.tools.NodeTool.NodeToolCmd;
+import org.apache.cassandra.tools.nodetool.layout.CassandraUsage;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Parameters;
+
+import static org.apache.cassandra.tools.nodetool.CommandUtils.parseOptionalKeyspace;
+import static org.apache.cassandra.tools.nodetool.CommandUtils.parseOptionalTables;
+import static org.apache.cassandra.tools.nodetool.CommandUtils.concatArgs;
 
 @Command(name = "disableautocompaction", description = "Disable autocompaction for the given keyspace and table")
-public class DisableAutoCompaction extends NodeToolCmd
+public class DisableAutoCompaction extends AbstractCommand
 {
-    @Arguments(usage = "[<keyspace> <tables>...]", description = "The keyspace followed by one or many tables")
+    @CassandraUsage(usage = "[<keyspace> <tables>...]", description = "The keyspace followed by one or many tables")
     private List<String> args = new ArrayList<>();
 
+    @Parameters(index = "0", description = "The keyspace name", arity = "0..1")
+    private String keyspace;
+
+    @Parameters(index = "1..*", description = "The table names", arity = "0..1")
```

Comment:
why is arity 0..1 ?

Reply:
`0..1` makes the argument not mandatory and it preserves backward compatibility, when the tables are empty. 
All `tables` withing keyspace are used in this case.

Reply:
Fixed.